devtoolsNG/basic/index.html

```diff
@@ -0,0 +1,36 @@
+<body>
+  <div id="log"></div> 
+  <input type="button" onclick="clearEntries()" value="Clear">
+ <script>
+
+   const websocket = new WebSocket('ws://localhost:12345');
+
+   websocket.onopen = e => {
+    websocket.send('WebSocket Open');  
+   };
+
+   websocket.onmessage = msg => {
+      appendEntry(msg.data)

```

Comment:
5 spaces of indent instead of 4, below you need one less space as well

Reply:
I think I fixed it all...